common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -230,13 +235,18 @@ private static MojoFrame doScore(MojoFrame requestFrame) {
     return responseFrame;
   }
 
-  private static MojoFrame doShapleyContrib(MojoFrame requestFrame) {
+  private static MojoFrame doShapleyContrib(MojoFrame requestFrame, boolean isOriginal) {
     log.debug(
             "Input has {} rows, {} columns: {}",
             requestFrame.getNrows(),
             requestFrame.getNcols(),
             Arrays.toString(requestFrame.getColumnNames()));
-    MojoFrame shapleyResponseFrame = pipelineShapley.transform(requestFrame);
+    MojoFrame shapleyResponseFrame;
+    if (isOriginal) {
+      shapleyResponseFrame = pipelineOriginalShapley.transform(requestFrame);
```

Comment:
regarding my earlier statement about loading mojos. Given that we are scoring in sequence: 
1. score predictions
2. score shapley if requested
3. etc.
Why don't we just do:
```
pipeline.setShapPredContrib(true);
MojoFrame shapleyResponseFrame = pipeline.transform(requestFrame);
pipeline.setShapPredContrib(false);
```

Reply:
Currently we cannot change the boolean on the runtime, unfortunately the pipeline doesn't support this
https://github.com/h2oai/mojo2/blob/7a1ab76b09f056334842a5b442ff89859aabf518/doc/shap.md
> either SHAP values or predictions can be provided, not both at the same time (with the same pipeline)

Reply:
yea nothing says we cannot set the boolean back to `false` afterwards. @Rajimut

Reply:
I tested it, reversing doesn't seem to work

Reply:
as far as I can tell, its just a setter. See here:
https://github.com/h2oai/mojo2/blob/8c7a253b1a6c886eca2be51b3e6eaac19a69e158/java/mojo2-runtime-impl/src/main/java/ai/h2o/mojos/runtime/MojoPipelineProtoImpl.java#L198-L205
and
https://github.com/h2oai/mojo2/blob/8c7a253b1a6c886eca2be51b3e6eaac19a69e158/java/mojo2-runtime-impl/src/main/java/ai/h2o/mojos/runtime/MojoPipelineProtoImpl.java#L110-L135

Reply:
😭 yea I tried it my self. Raises exception